tests/ProxyManagerTest/Factory/LazyLoadingValueHolderFactoryTest.php

```diff
@@ -72,12 +72,10 @@ protected function setUp() : void
      */
     public static function testWithOptionalFactory() : void
     {
-        $factory = new LazyLoadingValueHolderFactory();
-
-        $configuration = Assert::readAttribute($factory, 'configuration');
-
-        self::assertNotEmpty($configuration);
-        self::assertInstanceOf(Configuration::class, $configuration);
+        self::assertInstanceOf(
+            Configuration::class,
+            Assert::readAttribute(new LazyLoadingValueHolderFactory(), 'configuration')
```

Comment:
I first thought that `readAttribute` had something to do with `Assert`ion.

Reply:
`Assert` is something I added mytself specifically to work around the lack of assertions on state in newer PHPUnit versions